### PR TITLE
Test setup continued

### PR DESCRIPTION
**Pull request by metlos**
### What does this PR do?
This converts the rest of the integration tests to the new Test Setup logic.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-198
https://github.com/redhat-appstudio/service-provider-integration-operator/pull/348

Note that this builds on top of #348 so the diff is rather large now but should be much smaller once #348 is merged.

### How to test this PR?
no new logic, just changes in the integration tests

### CODE REVIEW

**Comment by metlos** [comment]
converted back to draft so that we indicate better this needs to wait until #348 is merged.

**Review by mshaposhnik — APPROVED** [review]

**Pull request merged**
